Tella/Scenes/Uwazi/ViewModel/TemplateCardViewModel.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TemplateCardViewModel.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 9/23/23.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+class TemplateCardViewModel {
+    var id = UUID()
```

Comment:
What do you think if you use the template id (if exist) instead of creating a new id (UUID())